jsonnet/prometheus-operator/thanosrulers-crd.json

```diff
@@ -3,8 +3,7 @@
   "kind": "CustomResourceDefinition",
   "metadata": {
     "annotations": {
-      "controller-gen.kubebuilder.io/version": "v0.16.3",
-      "operator.prometheus.io/version": "0.77.1"
```

Comment:
Looks like new update causing to remove this across CRDs

Reply:
The actual command shouldn't have been in an `else` clause, fixed. :)

Reply:
I verified that the removal is happening before the script is being executed.

```
┌[rexagod@nebuchadnezzar] [/dev/ttys000] [typo-27250] 
└[~/repositories/work/prometheus-operator]> cd pkg/apis/monitoring && /Users/rexagod/repositories/work/prometheus-operator/tmp/bin/controller-gen crd:crdVersions=v1 paths=./v1/. paths=./v1alpha1/. output:crd:dir=/Users/rexagod/repositories/work/prometheus-operator/example/prometheus-operator-crd/
┌[rexagod@nebuchadnezzar] [/dev/ttys000] [typo-27250 ⚡] 
└[~/repositories/work/prometheus-operator/pkg/apis/monitoring]> gss
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_alertmanagerconfigs.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_probes.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_prometheusagents.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_scrapeconfigs.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
 M ../../../example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
 ```

Reply:
Ah! It seems we don't use `operator.prometheus.io/version` [anymore](https://github.com/prometheus-operator/prometheus-operator/pull/6951/files#diff-127418fc75870b979eb861d8a41ff5ee719d8b093c485745fba6afcb901d073d)?